### DISCUSSION TITLE
Collaboration

### DISCUSSION THREAD

**Discussion by manaxtech** [Cyfrin/foundry-full-course-cu#436] [General]
I am learning web3 and solidity here. Is there anyone who want to learn this course together, to learn more efficiently and share knowledge?

**Comment by shikhar229169** [comment]
Hey, whenever you are stuck or have any query, just post your doubt in the discussions, we will be happy to answer those.

**Reply by manaxtech** [reply]
OK, and thanks a lot.

**Reply by infinite969** [reply]
Hey, Mana! I'm working through part 4, where are you at in the course? I am interested in collaborative learning. Shoot me a dm on discord! my username is: infinite96

**Comment by john-lawniczak** [comment]
I have been compiling everything I've learned into a repository: [Smart Contract Resources](https://github.com/the-vegetarian-vampire/Smart-Contract-Resources)

If you search around there is a ton of information out there. You can find yourself in an array of different rabbit holes.

**Reply by manaxtech** [reply]
Thanks for the reply and the information and I will definitely checkout your repo.
